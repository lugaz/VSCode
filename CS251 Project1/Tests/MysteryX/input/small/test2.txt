insertFront 838
deleteRear
getRear
deleteRear
getFront
deleteRear
deleteRear
deleteFront
deleteFront
deleteFront
insertFront 581
insertFront 571
deleteFront
getFront
deleteFront
getFront
getRear
deleteFront
deleteRear
deleteFront
insertFront 170
deleteFront
deleteRear
deleteFront
insertFront 663
deleteRear
deleteFront
deleteFront
insertFront 0
deleteRear
getRear
getRear
deleteRear
insertRear 523
deleteRear
deleteRear
deleteFront
getFront
insertRear 871
