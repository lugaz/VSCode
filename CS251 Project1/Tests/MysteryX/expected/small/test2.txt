838
-1
-1
-1
-1
-1
-1
-1
-1
571
581
581
-1
-1
-1
-1
-1
170
-1
-1
663
-1
-1
0
-1
-1
-1
523
-1
-1
-1
